content/_layouts/post.html.haml

```diff
@@ -42,6 +41,3 @@ layout: default
               = display_author_for(page)
 
         = content
-
-      %div#sidebar.col-md-4.events.vertical
-        = partial('sidebar.html.haml')
```

Comment:
Sorry about the signature - gets added when I reply on mobile and I know it's bad form.